Composer/packages/client/package.json

```diff
@@ -77,6 +77,7 @@
     "react-timeago": "^4.4.0",
     "recoil": "^0.0.13",
     "sanitize-html": "2.3.3",
+    "sinon": "^11.1.1",
```

Comment:
We dont need sinon for fakeTimers. Use jest fake timers. 
https://jestjs.io/docs/timer-mocks

Reply:
Revert the debounce function